extra/modules/greenbids-real-time-data/src/test/java/org/prebid/server/hooks/modules/greenbids/real/time/data/v1/GreenbidsRealTimeDataModuleTest.java

```diff
@@ -0,0 +1,14 @@
+package org.prebid.server.hooks.modules.greenbids.real.time.data.v1;
+
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class GreenbidsRealTimeDataModuleTest {
```

Comment:
I believe we can drop this one, it's kinda redundant

Reply:
removed